src/test/java/org/symphonyoss/integration/webhook/jira/parser/IssueUpdatedJiraParsetTest.java

```diff
@@ -114,4 +114,30 @@ public void testIssueUpdatedWithoutChangelogStatus() throws IOException, JiraPar
 
     assertEquals(expected, result);
   }
+
+  @Test
+  public void testIssueUpdatedEpicToNull() throws IOException, JiraParserException {
+    ClassLoader classLoader = getClass().getClassLoader();
+    Map<String, String> parameters = new HashMap<>();
+
+    JsonNode node = JsonUtils.readTree(classLoader.getResourceAsStream(EPIC_FILENAME));
```

Comment:
Better just copying the payload file and set the null value there instead of adding logic to the test here.